eventHandlers.sample.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * NOTE: Don't modify this file! Copy this file to `evenHandlers.js` and it will
+ *   automatically be included in the pool server!
+ */
+
+/**
+ * Fired when a REGISTER message is received, before creating a corresponding

```

Comment:
The description is misleading. The `PoolAgent` was already created at this time (because it is created before the first register arrives). There is also no reason why the `PoolAgent` instance is not a param on this function.

My suggestions would be
a) add `PoolAgent` param to `beforeRegister`
b) rename `beforeRegister` to `onRegisterMessage`
c) fix the description to state that it is fired before the message is processed by the PoolAgent
d) also rename `onRegister` to `onRegistrationCompleted` to make clear that it is fired after the registration.